pkgs/by-name/mi/microsoft-identity-broker/package.nix

```diff
@@ -60,13 +53,18 @@ stdenv.mkDerivation rec {
     classpath="$classpath:${jnr-posix}/share/java/jnr-posix-${jnr-posix.version}.jar"
     mkdir -p $out/bin
     makeWrapper ${openjdk11}/bin/java $out/bin/microsoft-identity-broker \
-      --add-flags "-classpath $classpath com.microsoft.identity.broker.service.IdentityBrokerService" \
-      --add-flags "-verbose"
-    makeWrapper ${openjdk11}/bin/java $out/bin/microsoft-identity-device-broker \
+      --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath buildInputs}" \
+      --add-flags "-classpath $classpath" \
+      --add-flags "-Xmx128m -Xss256k -XX:+UseParallelGC -XX:ParallelGCThreads=1" \
       --add-flags "-verbose" \
+      --add-flags "com.microsoft.identity.broker.service.IdentityBrokerService"
+
+    makeWrapper ${openjdk11}/bin/java $out/bin/microsoft-identity-device-broker \
+      --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath buildInputs}" \
       --add-flags "-classpath $classpath" \
-      --add-flags "com.microsoft.identity.broker.service.DeviceBrokerService" \
-      --add-flags "save"
```

Comment:
This I removed as the original package doesn't seem to have this argument passed in to `microsoft-identity-device-broker` either.